RNA-seq/01-qc_trim_quant.md

```diff
@@ -0,0 +1,244 @@
+
+There is no "one size fits all" approach for project organization. 
+It's most important that it's consistent, easy for you and others to find the files you need quickly, and minimizes the likelihood for errors (e.g., writing over files accidentally).
+
+### Quality control with FastQC
+
+The first thing our script does is use [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) for quality control in command line mode.
+Here's a link to the FastQC documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
+
+**Let's take a look at some example reports from the authors of FastQC:**
+
+* [Good Illumina data example report](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/good_sequence_short_fastqc.html) from FastQC
+* [Bad Illumina data example report](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/bad_sequence_fastqc.html) from FastQC
+
+FastQC runs a series of quality checks on sequencing data and provides an HTML report. As the authors point out in the [docs](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/2%20Basic%20Operations/2.2%20Evaluating%20Results.html):
+
+> It is important to stress that although the analysis results appear to give a pass/fail result, these evaluations must be taken in the context of what you expect from your library.
+
+The [documentation for individual modules/analyses](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/3%20Analysis%20Modules/) in FastQC is a great resource!
+
+To save time, our script only runs one FASTQ file for SRR585570 with the following command:
+
+```bash
+# In the interest of time, we'll run one of the fastq files through FastQC
+fastqc data/fastq/gastric_cancer/SRR585570/SRR585570_1.fastq.gz \
+	-o QC/gastric_cancer/fastqc_output
+```
+
+#### `-o`
+
+The `-o` flag allows us to specify where the output of FastQC is saved. 
+Note that this is saved in a different place than the raw data files and in a directory specifically for quality control information.
+
+For comparison, we've prepared a FastQC report for one of the sets of reads for another sample in the experiment.
+It can be found at `QC/gastric_cancer/fastqc_output/SRR585574_1_fastqc.html`.
+
+**Let's look at the results together.**
+
+### Preprocessing with fastp
+
+We use [fastp](https://github.com/OpenGene/fastp) to preprocess the FASTQ files ([Chen et al. _Bioinformatics._ 2018.](https://doi.org/10.1093/bioinformatics/bty560)). 
+Note that fastp has quality control functionality and many different options for preprocessing (see [all options on Github](https://github.com/OpenGene/fastp/blob/master/README.md#all-options)), most of which we will cover.
+Here, we focus on adapter trimming, quality filtering, and length filtering.
+
+Below, we discuss the two commands we used in the script.
+
+**Create a directory**
+
+```bash
+# Create a directory to hold the JSON and HTML output from Fastp
```

Comment:
People (as you have mentioned before) might breeze over these comments at the beginning of these code chunks that are actually explaining the whole purpose of the code chunk. You may want to consider in this context, moving these two code chunk's comments out of the code chunk and instead prefacing the code chunks. This way people know what they are looking for and are less likely (but probably still likely) to glaze over the entire chunk.

Reply:
It says **Create a directory** at line 97